types/aws-param-store/index.d.ts

```diff
@@ -27,17 +27,17 @@ export function getParametersSync(
 ): SSM.Types.GetParametersResult;
 
 export function getParametersByPath(
-    path: SSM.Types.ParameterNameList,
+    path: string,
     options?: SSM.Types.ClientConfiguration
 ): Promise<SSM.Types.ParameterList>;
 
 export function getParametersByPathSync(
-    path: SSM.Types.ParameterNameList,
+    path: string,
```

Comment:
This and the async version above can use `SSM.Types.PSParameterName` which ends up as string anyway but using the SDK types seems better to me.